rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java

```diff
@@ -337,4 +342,19 @@ static void setCacheDataChosenColumns(Configuration c, int chosen)
     {
         c.setInt(DATA_CACHE_TABLE_COLS_CHOSEN, chosen);
     }
+
+    public static int getBufferSize(Configuration c)
+    {
+        return c.getInt(bufferSizeConf, bufferSize);
+    }
+
+    public static int getLocalServerPort(Configuration conf)
+    {
+        return conf.getInt(localServerPortConf, 6666);
```

Comment:
extract out port number as a const variable in the start like others. And i would suggest to use 8898.